app/src/main/java/net/squanchy/schedule/view/ScheduleViewPagerAdapter.java

```diff
@@ -58,14 +59,15 @@ protected void bindView(ScheduleDayPageView view, int position) {
     @Override
     public CharSequence getPageTitle(int position) {
         LocalDateTime date = pages.get(position).getDate();
-        return date.toString(TITLE_FORMAT_TEMPLATE).toUpperCase();
+        return date.toString(TITLE_FORMAT_TEMPLATE).toUpperCase(Locale.getDefault());
```

Comment:
Those are for parsing, this is to format user-facing data, so it is correct, yes